new-site/gatsby-config.js

```diff
@@ -10,6 +10,34 @@ module.exports = {
         link: '/',
         subMenuLinks: [],
       },
+      {
+        name: 'Getting started',
+        link: '/getting-started',
+        subMenuLinks: [
+          {
+            name: 'Getting started',
+            link: '/getting-started',
+          },
+          {
+            name: 'Designer',
+            link: '/getting-started/designer',
+            withDivider: true,
+          },
+          {
+            name: 'Developer',
+            link: '/getting-started/developer',
+          },
+          {
+            name: 'FAQ',
```

Comment:
@minevala This does not work as expected. I would like to have only the link to the FAQ page and not links to Tab-pages below them. How can I fix this?

Reply:
It might try to find subpages with path. I need to check this.

Reply:
It is one level higher than our Component pages are. It probably would do the same in Component pages too but the navigation cannot display more levels than 2.

Reply:
The fragments require:
export default ({ children }) => <>{children}</>;

Reply:
> export default ({ children }) => <>{children}</>;

While this indeed makes the pages function as fragments, it does not affect the side navigation at all. Tabs are still part of the navigation but now they open as fragment pages without navigation and footer.

<img width="299" alt="image" src="https://user-images.githubusercontent.com/4214671/159458243-936dbaf5-0db6-43ad-9724-c35c2a1b3ed9.png">

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/4214671/159458278-00b64382-3082-479a-a2f5-f01e903ce141.png">